### DISCUSSION TITLE
FYI: Current Development Status (Temporary Hiatus)

### DISCUSSION THREAD

**Discussion by Mezryss** [Mezryss/FVTT-Genesys#63] [Announcements]
I've mentioned this in both Discords (Genesys and Foundry), but I figured I should post here as well to let folks know that I'm currently taking a hiatus from working on Genesys. Truth be told, Genesys was only ever a project I was building with the goal of refreshing my memory for Foundry dev before I began working on my actual goal of building a system for Infinity.

While I've certainly learned a lot and had a lot of fun building Genesys, and I do intend to come back to it, the project really snowballed to a point where it's become exhausting to work on (especially thanks to some of the decisions I made while building it, since all of my Vue work was still very experimental and exploratory).

That, combined with general life issues, wanting to start working on Infinity, and just the sheer frequency with which I've seen pro-AI (particularly ChatGPT) discussions in the spaces where I was discussing this system, just all built up into the project not being that fun for me to focus on anymore.

While I'm on this hiatus, I do plan to continue accepting reasonable-looking PRs and will likely build new releases on Fridays to make the changes in these PRs available to users of the system. I'm also open to people doing PRs for feature work, if they'd like to tackle any of the current feature development items during this period.

I do plan to come back to this project at some point, but I'm not sure when that might be or what changes I might want to introduce when I do, as I've already made some significant strides in finding a much cleaner development workflow for Vue in Foundry.

**Comment by FilBot3** [comment]
Thank you for this update! I was just going to post asking if this system was ready for FoundryVTT v11. This answers that!

**Reply by FilBot3** [reply]
Nevermind! I was wrong!
https://foundryvtt.com/packages/genesys
